py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -89,16 +89,47 @@ def aten_ops_batch_norm(
     )
 
 
+@dynamo_tensorrt_converter(
+    torch.ops.aten._native_batch_norm_legit_no_training.default,
+    capability_validator=one_user_validator,
+)  # type: ignore[misc]
+def aten_ops_batch_norm_legit_no_training(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.normalization.batch_norm(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        input=args[0],
+        weight=args[1],
+        bias=args[2],
+        running_mean=args[3],
+        running_var=args[4],
+        training=False,
+        momentum=args[5],
+        eps=args[6],
+        cudnn_enabled=args_bounds_check(args, 8, True),
```

Comment:
`cudnn_enabled` is not used in this converter according to the schema, so it can be directly set to False.